CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 - Add master functions TerminateSlaves, CancelSlaves, Quit
 
+## 0.3.2 (2024-03-11)
+
+### Changes
+
+- When doing manual picking, propagate packFormationParameters to planning (instead of dynamicGoalGeneratorParameters).
```

Comment:
```suggestion
- [BREAKING] `ManuallyPlacePackItem` now takes the input argument `packFormationParameters` instead of `dynamicGoalGeneratorParameters`
```